src/Arcus.Messaging.Pumps.ServiceBus/Extensions/MessageCorrelationInfoExtensions.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Arcus.Messaging.Abstractions;
+using GuardNet;
+using Microsoft.Azure.ServiceBus;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace Arcus.Messaging.Pumps.ServiceBus.Extensions
+{
+    /// <summary>
+    /// Azure ServiceBus plugin to add correlation information to the inbound and outbound messages.
+    /// </summary>
+    public static class MessageCorrelationInfoExtensions
```

Comment:
Since the extensions are on `Message`, shouldn't we move this to `MessageExtensions`?